### PR TITLE
Rename references to addressbook

### PR DESCRIPTION
**Pull request by tobihy**
Refactor all references to AddressBook in code into PetBook.

This includes methods and Javadoc comments as well.

### CODE REVIEW

File: src/test/java/woofareyou/storage/JsonPetBookStorageTest.java [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+        assertThrows(DataConversionException.class, () -> readPetBook("invalidPetsPetBook.json"));
+    }
+
+    @Test
+    public void readPetBook_invalidAndValidPetBook_throwDataConversionException() {
+        assertThrows(DataConversionException.class, () -> readPetBook("invalidAndValidPetBook.json"));
+    }
+
+    @Test
+    public void readAndSavePetBook_allInOrder_success() throws Exception {
```
Comment (Tiffanylin21) [inline src/test/java/woofareyou/storage/JsonPetBookStorageTest.java:64]: This test has a lot of `AddressBook`!

File: src/test/java/woofareyou/storage/JsonPetBookStorageTest.java [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+        // Save and read without specifying file path
+        original.addPet(IDA);
+        jsonAddressBookStorage.savePetBook(original); // file path not specified
+        readBack = jsonAddressBookStorage.readPetBook().get(); // file path not specified
+        assertEquals(original, new PetBook(readBack));
+
+    }
+
+    @Test
+    public void saveAddressBook_nullAddressBook_throwsNullPointerException() {
```
Comment (Tiffanylin21) [inline src/test/java/woofareyou/storage/JsonPetBookStorageTest.java:90]: Same with this!

File: src/test/java/woofareyou/storage/JsonPetBookStorageTest.java [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+        assertThrows(NullPointerException.class, () -> saveAddressBook(null, "SomeFile.json"));
+    }
+
+    /**
+     * Saves {@code addressBook} at the specified {@code filePath}.
+     */
+    private void saveAddressBook(ReadOnlyPetBook addressBook, String filePath) {
```
Comment (Tiffanylin21) [inline src/test/java/woofareyou/storage/JsonPetBookStorageTest.java:97]: Same with this too!

File: src/test/java/woofareyou/storage/JsonPetBookStorageTest.java [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+        try {
+            new JsonPetBookStorage(Paths.get(filePath))
+                    .savePetBook(addressBook, addToTestDataPathIfNotNull(filePath));
+        } catch (IOException ioe) {
+            throw new AssertionError("There should not be an error writing to the file.", ioe);
+        }
+    }
+
+    @Test
```
Comment (Tiffanylin21) [inline src/test/java/woofareyou/storage/JsonPetBookStorageTest.java:106]: And this ahahha

File: src/test/java/woofareyou/storage/JsonSerializablePetBookTest.java [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+                JsonSerializablePetBook.class).get();
+        assertThrows(IllegalValueException.class, dataFromFile::toModelType);
+    }
+
+    @Test
+    public void toModelType_sameNamePetsDifferentOwner_success() throws Exception {
+        JsonSerializablePetBook dataFromFile = JsonUtil.readJsonFile(SIMILAR_PET_FILE,
+                JsonSerializablePetBook.class).get();
+        PetBook petBookFromFile = dataFromFile.toModelType();
+        PetBook similarPetsPetBook = SimilarPets.getSimilarAddressBook();
```
Comment (Tiffanylin21) [inline src/test/java/woofareyou/storage/JsonSerializablePetBookTest.java:45]:
```suggestion
        PetBook similarPetsPetBook = SimilarPets.getSimilarPetBook();
```
Think need to refactor the above method from `getSimilarAddressBook()` to `getSimilarPetBook()`!

File: src/test/java/woofareyou/testutil/SimilarPets.java [hunk 6]
```diff
@@ -28,8 +28,8 @@ private SimilarPets() {} // prevents instantiation
     /**
      * Returns an {@code AddressBook} with all the similar pets.
      */
-    public static AddressBook getSimilarAddressBook() {
-        AddressBook ab = new AddressBook();
+    public static PetBook getSimilarAddressBook() {
```
Comment (Tiffanylin21) [inline src/test/java/woofareyou/testutil/SimilarPets.java:31]: need to refactor method! (same method mentioned in previous comment)

File: src/test/java/woofareyou/testutil/TypicalPets.java [hunk 7]
```diff
@@ -69,8 +69,8 @@ private TypicalPets() {} // prevents instantiation
     /**
      * Returns an {@code AddressBook} with all the typical pets.
```
Comment (Tiffanylin21) [inline src/test/java/woofareyou/testutil/TypicalPets.java:70]:
```suggestion
     * Returns an {@code PetBook} with all the typical pets.
```

**Review by Tiffanylin21 — COMMENTED** [review]
Great job overall, thank you for all the refactoring! Just spotted some minor naming and one more method to refactor :)

File: src/test/java/woofareyou/storage/JsonSerializablePetBookTest.java (hunk 5 above)
Comment (tobihy) [reply src/test/java/woofareyou/storage/JsonSerializablePetBookTest.java:45]: Yep!

**Pull request merged**